.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+## Overview
+
+< A brief description of the pull request >
+
+---
+
+| Q             | A
+| ------------- | ---
+| Bug fix?      | yes/no
+| New feature?  | yes/no
+| BC breaks?    | yes/no
+| Deprecations? | yes/no
+| Tests pass?   | yes/no
```

Comment:
Maybe a "Test" section toward the end? Or is it too much?

## Tests
*Does your PR passes the existing tests (Unit, E2E, BackstopJS)? Is it adding new ones to ensure comprehensive coverage?*